linux_os/guide/system/auditing/policy_rules/audit_rules_for_ospp/rule.yml

```diff
@@ -0,0 +1,52 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8
+
+title: 'Configure audit according to OSPP requirements'
+
+description: |-
```

Comment:
Description could use the same hash treatment of OCIL, so from it is obvious what is expected. Even better would be version, but those files in audit package are not internally versioned, so we have to go with the hash for now.

Reply:
Now, description doesn't mention hashes. It suggests to remediate by copy the pre-configured audit rules from `/usr/share/doc/audit/rules/`. And to check by using diff command.